pkgs/by-name/as/asfa/package.nix

```diff
@@ -0,0 +1,61 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  gawk,
+  help2man,
+  libssh2,
+  openssl,
+  pkg-config,
+  ...
```

Comment:
```suggestion
```

`pkgs.callPackage` is pretty smart and doesn't try to populate the attrset with everything if `...` is not used, so `...` can be removed`

Reply:
Good to know, adapted :+1: